profiler/src/ProfilerEngine/Datadog.Profiler.Native/CorProfilerCallback.cpp

```diff
@@ -116,6 +117,7 @@ bool CorProfilerCallback::InitializeServices()
     _pStackSnapshotsBufferManager = RegisterService<StackSnapshotsBufferManager>(_pThreadsCpuManager, _pSymbolsResolver);
 
     auto pWallTimeProvider = RegisterService<WallTimeProvider>(_pConfiguration.get(), _pFrameStore.get(), _pAppDomainStore.get());
+    auto pCpuTimeProvider = RegisterService<CpuTimeProvider>(_pConfiguration.get(), _pFrameStore.get(), _pAppDomainStore.get());
```

Comment:
use `auto*` to be more explicit/clear (even if the deduced type is a pointer)

Reply:
✅